tests/image_creation.sh

```diff
@@ -346,6 +390,49 @@ test_distro_alpine()
 	run_test "${name}" "" "alpine" "no" "init"
 }
 
+# Displays a list of all distro test functions
+get_distro_test_names()
+{
+	typeset -F | awk '{print $3}' |\
+		grep "^${test_func_prefix}" | sort
+}
+
+# Displays a list of distros which can be tested
+list_distros()
+{
+	local funcs
+
+	get_distro_test_names | sed "s/${test_func_prefix}//g"
+}
+
+test_single_distro()
+{
+	local -r distro="$1"
+
+	[ -z "$distro" ] && die "distro cannot be blank"
+
+	local -r expected_func="${test_func_prefix}${distro}"
+
+	local test_funcs
+	test_funcs=$(get_distro_test_names)
+	
+	local defined_func
+	defined_func=$(echo "$test_funcs" | grep "^${expected_func}$" || true)
+
+	if [ -z "$defined_func" ]
+	then
+		local distros
+
+		# make a comma-separated list
+		distros=$(list_distros | tr '\n' ',' | sed 's/,$//g')
```

Comment:
you could pull that out and add it as part of the help or as a param (`--list-distros`)

Reply:
Not sure I follow - this PR already includes a new `--list` option which lists the distros.

Reply:
Ah, so it does - I missed that. np then.